linera-views/src/backends/dynamo_db.rs

```diff
@@ -320,7 +327,8 @@ pub struct DynamoDbStoreInternal {
     namespace: String,
     semaphore: Option<Arc<Semaphore>>,
     max_stream_queries: usize,
-    root_key: Vec<u8>,
+    start_key: Vec<u8>,
+    root_key_written: Arc<Mutex<bool>>,
```

Comment:
Same here: Should probably be `AtomicBool`.

Reply:
Yes, that works.